packages/zowe-explorer-api/src/extend/IRegisterClient.ts

```diff
@@ -132,4 +133,6 @@ export interface IRegisterClient {
      * @returns the registered API instance
      */
     getCommonApi?(profile: imperative.IProfileLoaded): MainframeInteraction.ICommon;
+
+    getDatasetAttrProvider?(): DataSetAttributesProvider;
```

Comment:
(nitpick) the function name is `getDatasetsAttrProvider`, but the type is `DataSet...`:


```suggestion
    getDataSetAttrProvider?(): DataSetAttributesProvider;
```